Gems/PhysX/Code/Source/ArticulationLinkComponent.cpp

```diff
@@ -845,11 +854,96 @@ namespace PhysX
         }
         return AZ::Vector3::CreateZero();
     }
+
+    const AzPhysics::SimulatedBody* ArticulationLinkComponent::GetSimulatedBodyConst() const
+    {
+        const AZ::Entity* rootEntity = GetArticulationRootEntity();
+        const auto rootComponent = rootEntity->FindComponent<ArticulationLinkComponent>();
+
+        return AZ::Interface<AzPhysics::SceneInterface>::Get()->GetSimulatedBodyFromHandle(
+            rootComponent->m_attachedSceneHandle, GetSimulatedBodyHandle());
+    }
+
+    AzPhysics::SimulatedBody* ArticulationLinkComponent::GetSimulatedBody()
+    {
+        const AZ::Entity* rootEntity = GetArticulationRootEntity();
+        auto rootComponent = rootEntity->FindComponent<ArticulationLinkComponent>();
+
+        return AZ::Interface<AzPhysics::SceneInterface>::Get()->GetSimulatedBodyFromHandle(
+            rootComponent->m_attachedSceneHandle, GetSimulatedBodyHandle());
```

Comment:
This is exactly the same as `ArticulationLinkComponent::GetSimulatedBodyConst()`. Since the other function was done for the `const`-ness for `getAABB()` below, just return the results of that function here.
```suggestion
        return GetSimulatedBodyConst();
```